norsu/extension.py

```diff
@@ -1,20 +1,35 @@
 import os
 import shlex
+import shutil
 
 from pkg_resources import resource_filename
 
 from .config import CONFIG, TOOL_MAKE
 from .exceptions import Error
 from .terminal import Style
-from .utils import execute, ExecOutput
+from .utils import execute, ExecOutput, str_args_to_dict
 
 
 class Extension:
     def __init__(self, work_dir, pg_config):
         self.work_dir = work_dir
         self.pg_config = pg_config
 
-    def make(self, targets=None, options=None):
+        self.specs = None
+        specs_dir = os.path.join(work_dir, 'specs')
```

Comment:
We probably shouldn't hardwire `specs` dir name. Maybe we could add a default-able flag to the `pgxs` command?